Epsilon.Host.WebApi/Controllers/LearningController.cs

```diff
@@ -39,13 +39,9 @@ public async Task<ActionResult<LearningDomain>> GetDomain(string name)
     }
     
     [HttpGet("domain/{tenantId}/outcomes")]
-    public async Task<ActionResult<LearningDomainOutcome>> GetDomainOutcomes(string tenetId)
+    public async Task<ActionResult<LearningDomainOutcome>> GetDomainOutcomes(string tenantId)
```

Comment:
> We already discussed this in the meeting, but I'm just putting it here for reference.

Due to the fact that Fontys is our current only tenant, you guys may opt to not yet implement the filtering logic in the learning domain service to only get the outcomes for a specific tenant. What would be nice is to make an extra check to see if the tenant id matches the one from fontys, and if it doesn't return a not found or something along those lines. Otherwise just remove the tenant parameter for now.